src/test/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendRecorderTest.java

```diff
@@ -208,9 +210,10 @@ public void testSuccessfulBuildStatusFilterEmpty() throws Exception {
 
         given(VulnerabilityTrendHelper.createSDK(anyString(), anyString(), anyString(), anyString())).willReturn(contrastSDKMock);
 
-        TeamServerProfile profile = new TeamServerProfile("local", "contrast", "demo", "demo",
-                "www.google.com", "org-uuid",
-                false, Result.FAILURE.toString(), true);
+        TeamServerProfile profile = mock(TeamServerProfile.class);
```

Comment:
Why do we need to mock these now?

Reply:
@gmoore The TeamServerProfile constructor uses a running jenkins instance to setup encryption of the apiKey and serviceKey, so we would get an error if using the previous approach.